configuration/logiVehicle.sqf

```diff
@@ -33,6 +33,24 @@ SET_CUSTOM_NAME("Small Ammo Crate"); // This command lets you give a custom name
 ADD_VEHICLE_WITH_GEAR(example,"B_supplyCrate_F",5,"crate_med");
 SET_CUSTOM_NAME("Large Ammo Crate");
 
+
+// Special "Init" code, see below for details.
+private _tankCode = 
+{
+    params ["_spawnedVic", "_logiVic", "_logiType", "_faction", "_spawnedVicName"];
+
+    // Enable hunter-killer by default.
+    _spawnedVic setVariable ["ace_hunterkiller", true, true];
+
+    // Add a toolkit and backpack only.
```

Comment:
Consider clarifying the execution scope of the code (does it run on the globally/ on the server/wherever it is local?) - *most* people won't need this information, but it's good to have it right there for advanced users.